### PR TITLE
Update README.md and bump Collection version

### PR DESCRIPTION
**Pull request by sdodsley**
##### SUMMARY
Update README with new FA Files modules

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
README.md

### CODE REVIEW

File: collections/ansible_collections/purestorage/flasharray/README.md [hunk 1]
```diff
@@ -29,11 +29,14 @@ All modules are idempotent with the exception of modules that change or set pass
 - purefa_banner - manage the CLI and GUI login banner of the FlashArray
 - purefa_connect - manage FlashArrays connecting for replication purposes
 - purefa_console - manage Console Lock setting for the FlashArray
+- purefa_directory - manage FlashArray managed fire system directories
```
Comment (dnix101) [inline collections/ansible_collections/purestorage/flasharray/README.md:32]: fire = file

File: collections/ansible_collections/purestorage/flasharray/README.md [hunk 2]
```diff
@@ -47,6 +50,7 @@ All modules are idempotent with the exception of modules that change or set pass
 - purefa_phonehome - manage the phonehome setting for the FlashArray
 - purefa_pod - manage ActiveCluster pods in FlashArrays
 - purefa_pod_replica - manage ActiveDR pod replica links in FlashArrays
+- purefa_policy - manage FlashArray managed policies for NFS, SMB and snapshot
```
Comment (dnix101) [inline collections/ansible_collections/purestorage/flasharray/README.md:53]: snapshot**b**?

Comment (sdodsley) [reply collections/ansible_collections/purestorage/flasharray/README.md:53]: Yes - it will cover snapshot policy for filesystems as well

Comment (dnix101) [reply collections/ansible_collections/purestorage/flasharray/README.md:53]: I ment snapshots instead of snapshot?

**Review by dnix101 — APPROVED** [review]
shipit

**Pull request merged**